tests/console/validate_packages_and_patterns.pm

```diff
@@ -35,35 +35,20 @@ if (check_var('VALIDATE_PCM_PATTERN', 'azure')) {
     };
 }
 elsif (check_var('VALIDATE_PCM_PATTERN', 'aws')) {
-    # We have different pattern names on SLE 15 and SLE 12
-    my $aws_pattern = is_sle('15+') ? 'Amazon_Web_Services' : 'Amazon-Web-Services';
-    my @aws_pattern_packages = qw(
-      aws-cli cloud-init
-      cloud-regionsrv-client
-      docker-img-store-setup growpart supportutils-plugin-suse-public-cloud
-      regionServiceClientConfigEC2 s3fs
-    );
-    if (is_sle('15+')) {
-        push @aws_pattern_packages, qw(
-          cloud-regionsrv-client-plugin-ec2
-          patterns-public-cloud-15-Amazon-Web-Services
-          python3-ec2deprecateimg python3-ec2metadata python3-ec2publishimg
-          python3-ec2uploadimg python3-s3transfer python3-susepubliccloudinfo
-        );
-    }
-    else {
-        push @aws_pattern_packages, qw(
-          patterns-public-cloud-Amazon-Web-Services
-          python-ec2deprecateimg python-ec2metadata python-ec2publishimg
-          python-ec2uploadimg python-s3transfer python-susepubliccloudinfo
-        );
-    }
-    $software{$aws_pattern} = {
+    # Different pattern and packages names on SLE 15 and SLE 12
+    my %aws_specific = is_sle('15+') ? (name => 'Amazon_Web_Services', sle_version => '15-', py_version => 'python3') :
+      (name => 'Amazon-Web-Services', py_version => 'python');
+    $software{%aws_specific{name}} = {
```

Comment:
this is not valid perl - I wonder how you validated this, but your validation run is unreachable

Reply:
I assume you came to this PR due to broken tests? Feel free to revert the change in a revert PR and ping the author to bring back the changes properly

Reply:
5794 - @jknphy already has email from @msmeissn

Reply:
Thanks for the feedback and the revert and sorry for the inconvenience.
I can see in the logs of the validations both errors, but they didn't produce a failed job on my local machine neither detected by Travis. Is there something in my configuration that I should tweak to avoid this in the future?
New PR: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/5795

Reply:
It's very possible that the old sle12 perl is different - why travis didn't catch it, I don't know